lib/std/debug.zig

```diff
@@ -666,158 +666,160 @@ pub fn openSelfDebugInfo(allocator: *mem.Allocator) anyerror!DebugInfo {
 
 /// TODO resources https://github.com/ziglang/zig/issues/4353
 fn openCoffDebugInfo(allocator: *mem.Allocator, coff_file_path: [:0]const u16) !ModuleDebugInfo {
-    const coff_file = try std.fs.openFileAbsoluteW(coff_file_path.ptr, .{});
-    errdefer coff_file.close();
+    noasync {
+        const coff_file = try std.fs.openFileAbsoluteW(coff_file_path.ptr, .{});
```

Comment:
In order to use `noasync` here, the program must guarantee that `openFileAbsoluteW` will not block. What makes this guaranteed? I think `openFileAbsoluteW` is missing an option in the `OpenFlags` argument which would provide this guarantee.

Reply:
`noasync` asserts that it won't `suspend` right? On Windows opening a file is always blocking and when opening the file one has to specify whether or not next usages (like write, reads) are async or not, using the `create_options` flags.